<?xml version="1.0"?>
<launch>
  <!-- group for ego racecar -->
  <group ns="ego_racecar">
      <!-- Open the model file -->
      <arg name="racecar_xacro" default="$(find f1tenth_gym_ros)/ego_racecar.xacro"/>
      <param name="tf_prefix" value="ego_racecar"/>
      <param name="robot_description" command="xacro --inorder '$(arg racecar_xacro)'"/>
      <!-- Add the transformations -->
      <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
  </group>
  <!-- group for opponent racecar -->
  <group ns="opp_racecar">
      <!-- Open the model file -->
      <arg name="racecar_xacro" default="$(find f1tenth_gym_ros)/opp_racecar.xacro"/>
      <param name="tf_prefix" value="opp_racecar"/>
      <param name="robot_description" command="xacro --inorder '$(arg racecar_xacro)'"/>
      <!-- Add the transformations -->
      <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
  </group>
</launch>
